### PR TITLE
LP-2676 Write technical documentation for webinars app

### PR DESCRIPTION
**Pull request by harris-irfan**
Ticket: [LP-2676](https://philanthropyu.atlassian.net/browse/LP-2676)

### CODE REVIEW

**Comment by omnipreneurshipA** [comment]
## Diff Coverage Report - ADG
Check-out complete **[report](https://app.circleci.com/pipelines/github/OmnipreneurshipAcademy/edx-platform/1665/workflows/workflows_id/jobs/1665/artifacts)**. Coverage !!! :rocket:
```
-------------
Diff Coverage
Diff: origin/develop...HEAD, staged and unstaged changes
-------------
No lines with coverage information in this diff.
-------------
```

**Review by javeriaejaz-arbisoft — APPROVED** [review]

File: openedx/adg/lms/webinars/__init__.py [hunk 1]
```diff
@@ -1,5 +1,41 @@
...
+This feature enables ADG admins to:
+ - create new events(the words 'event' and 'webinar' are used interchangeably) on the admin site
+ - update existing events
+ - cancel existing events
+ - clone existing events
+ - view all events i.e. upcoming, cancelled and delivered
+ - view list of registered users against events
+
+While creating a new event, admins can:
+ - add event details like title, schedule, description, banner image, external meeting link for webinar
```
Comment (ahmed-faraz46) [inline openedx/adg/lms/webinars/__init__.py:16]: Let's not dive to such details that some small change in code would require a change in documentation. I guess Admins can add, update, cancel, clone existing events (The above paragraph) is enough for the purpose of technical documentation

**Review by ahmed-faraz46 — APPROVED** [review]
Just one suggestion, rest looks good 👍

File: openedx/adg/lms/webinars/__init__.py (hunk 1 above)
Comment (ahmed-faraz46) [reply openedx/adg/lms/webinars/__init__.py:16]: .

**Review by ahmed-faraz46 — CHANGES_REQUESTED** [review]

**Comment by omnipreneurshipA** [comment]
## Diff Coverage Report - ADG
Check-out complete **[report](https://app.circleci.com/pipelines/github/OmnipreneurshipAcademy/edx-platform/1667/workflows/workflows_id/jobs/1667/artifacts)**. Coverage !!! :rocket:
```
-------------
Diff Coverage
Diff: origin/develop...HEAD, staged and unstaged changes
-------------
No lines with coverage information in this diff.
-------------
```

**Review by ahmed-faraz46 — APPROVED** [review]

**Review by asadali145 — APPROVED** [review]

**Pull request merged**